web_app/api/user.py

```diff
@@ -42,10 +42,9 @@ async def get_user_contract(wallet_id: str) -> str:
     user = user_db.get_user_by_wallet_id(wallet_id)
     if user is None:
         raise HTTPException(status_code=404, detail="User not found")
-    elif not user.is_contract_deployed:
+    if not user.is_contract_deployed:
```

Comment:
Please, if you use chatgpt, not remove code which is not related to your task

Reply:
> Please, if you use chatgpt, not remove code which is not related to your task

Hello sir. I didnt remove the code intentionally. I modified them because of pylint. Pylint was displaying error regarding some code and i modified them for pylint test to pass.